fakestorage/object.go

```diff
@@ -763,8 +764,45 @@ func (s *Server) downloadObject(w http.ResponseWriter, r *http.Request) {
 	var content io.Reader
 	content = obj.Content
 	status := http.StatusOK
-	ranged, start, lastByte, satisfiable := s.handleRange(obj, r)
-	contentLength := lastByte - start + 1
+
+	transcoded := false
+	ranged := false
+	start := int64(0)
+	lastByte := int64(0)
+	satisfiable := true
+	contentLength := int64(0)
+
+	isHandledTranscoding := func() bool {
```

Comment:
nit: we could call this just `handledTranscoding` or `hasHandledTranscoding`
```suggestion
	handledTranscoding := func() bool {
```

Reply:
Fixed by https://github.com/fsouza/fake-gcs-server/pull/924/commits/23990176b9cde0ea005b5a3a48f38f19b08bef0b